UrologicalSupplies/R4/files/UrologicalSuppliesPrepopulation-0.1.0.cql

```diff
@@ -0,0 +1,128 @@
+library UrologicalSuppliesPrepopulation version '0.1.0'
+using FHIR version '4.0.0'
+include FHIRHelpers version '4.0.0' called FHIRHelpers
+
+codesystem "ICD-10": 'http://hl7.org/fhir/sid/icd-10'
+codesystem "ICD-10-CM": 'http://hl7.org/fhir/sid/icd-10-cm'
+codesystem "LOINC": 'http://loinc.org'
+codesystem "SNOMED-CT": 'http://snomed.info/sct'
+codesystem "HCPCS": 'https://bluebutton.cms.gov/resources/codesystem/hcpcs'
+
+valueset "Urinary Incontinence or Chronic Urinary Retention Disorder": 'http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1219.70'
+valueset "Urinary Collection and Irrigation System Device or Supply": 'http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1219.49'
+valueset "Administrative Gender (HL7 V3)": 'http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113883.1.11.1'
+valueset "Urinary Collection and Irrigation System Device or Supply	": 'http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1219.61'
+valueset "Intermittent Urinary Catheter or Urological Supply for Intermittent Urinary Catheter Supply": 'http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1219.59'
+valueset "External Catheter or Urinary Collection Device": 'http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1219.62'
+valueset "Miscellaneous Urinary Catheter Supply": 'http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1219.63'
```

Comment:
I don't see you define those valueset in the Library-R4-Urologicoxxx.json. You need to put those valuesets as "dataRequirement".